src/Craftify.Geometry.Domain/Arc.cs

```diff
@@ -1,3 +1,43 @@
 ﻿namespace Craftify.Geometry.Domain;
 
-public record Arc(Point3D Center, double Radius, double StartAngle, double EndAngle) : Curve;
+public static class Arc
+{
+    public static ArcSegment ByCenterPointRadiusSweepAngle(
+        Point3D center,
+        double radius,
+        double startAngle,
+        double sweepAngle,
+        Vector3D normal
+    )
+    {
+        return new(
+            center,
+            radius,
+            startAngle.NormalizeAngle(),
+            sweepAngle.NormalizeAngle(),
+            normal
+        );
+    }
+
```

Comment:
Do we need one more method CreateByThreePoints(Point3D start, Point3D end, Point3D pointOnLine);

Reply:
It will be done in the future.
The plan is to add all the methods for creating arcs that we have in Dynamo.